File: CSharp/Ejercicios/Ejercicio1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
namespace ejercicio1
{
    class Program
    {
        static void Main(string[] args)
        {
            int promedio = 0;
            int[] notas = new int[10];
            for (int i = 0; i < notas.Length; i++)
            {
                Console.Write("Ingrese la nota: " + (i + 1));
                string nota = Console.ReadLine();
                notas[i] = int.Parse(nota);
                promedio += notas[i];
            }
            promedio = promedio / 10;
            Console.WriteLine("El promedio es: " + promedio);
        }
    }
}

File: Java LN/Logica_JAVA/Ejercicios_clase_21_de_Abril/Ejercicio1.java
import javax.swing.JOptionPane;

public class Ejercicio1{
    public static void main(String[] args){
        float promedio = 0;
        int [] notas = new int[10];
        for(int i = 0; i < notas.length; i++)
        {
            notas[i] = Integer.parseInt(JOptionPane.showInputDialog("Ingrese la nota " + (i+1)));
            promedio += (float)notas[i];
        }
        promedio = promedio / notas.length;
        System.out.println("El promedio es: " + promedio);
    }
}
